src/ProjectTemplates/Web.ItemTemplates/content/RazorComponent/.template.config/template.json

```diff
@@ -0,0 +1,44 @@
+{
+  "$schema": "http://json.schemastore.org/template",
+  "author": "Microsoft",
+  "classifications": [
+    "Web",
+    "ASP.NET"
+  ],
+  "name": "Razor Component",
```

Comment:
@danroth27, are we still calling the item template "Razor Component" now that we are renaming to Blazor server-side?

Reply:
cc @javiercn who is doing the rename.

Reply:
I think so.

Reply:
Yes, .razor files are still Razor components.